go/tendermint/apps/scheduler/scheduler.go

```diff
@@ -569,37 +569,26 @@ func (app *schedulerApplication) electValidators(ctx *abci.Context, beacon []byt
 		if err := entityStake.checkThreshold(n.EntityID, staking.KindValidator, false); err != nil {
 			continue
 		}
-		preFilteredNodeList = append(preFilteredNodeList, n)
+		nodeList = append(nodeList, n)
 		entMap[n.EntityID.ToMapKey()] = true
+
+		// Everyone that "runs" a validator node is eligible for rewards.
+		//
+		// Yes, this is stupid in that, there's no proof that the node is
+		// actually running.
```

Comment:
Yeah so even if the node is actually scheduled and it is not running it will not be penalized in any way (except not getting fees) as we are not slashing for liveness. So this is just distributing rewards to people that have registered "nodes" and have some stake bonded.

Reply:
I stand by "this is stupid".